www/src/pages/components/button.mdx

```diff
@@ -21,51 +22,125 @@ import SingleComponentStatus from '../../components/SingleComponentStatus';
   }
 />
 
+<p style={{ fontSize: '1.5em' }}>
+  This is a pass through component from React-Bootstrap.<br/>
+  <a href="https://react-bootstrap.github.io/components/cards/" target="_blank" rel="noopener noreferrer">
+    See React-Bootstrap for documentation.
+  </a>
+</p>
+
+##### Basic Usage
```

Comment:
Do we need to maintain this documentation for `Button` now that it's from react-bootstrap?

Reply:
The react-bootstrap documentation here is just for ease of consumption so we're not kicking devs out to external docs for the majority need.